src/test/kfs_rest_parser_test.cpp

```diff
@@ -319,6 +319,98 @@ TEST_F(KFSRestParserTest, parseValidRequestBYTES) {
     ASSERT_EQ(binary_data_size_parameter->second.int64_param(), 4);
 }
 
+TEST_F(KFSRestParserTest, parseValidRequestBYTES_noBinaryDataSizeParameter) {
+    std::string request = R"({
+    "inputs" : [
+        {
+        "name" : "input0",
+        "shape" : [ 2, 2 ],
+        "datatype" : "BYTES"
+        }
+    ]
+    })";
```

Comment:
Is this correct behavior? Sending binary tensor with shape [2,2] and no binary data?